packages/blade/src/components/Modal/_decisions/decisions.md

```diff
@@ -0,0 +1,152 @@
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Content of the Modal | ✅       |
+
+### `ModalHeader` API
+> `ModalHeader` API will be similar to [`BottomSheetHeader` API](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/BottomSheet/_decisions/decisions.md#bottomsheetheader-api) with the exception of `showBackButton` & `onBackButtonClick` props.
+
+| Prop      | Type            | Default     | Description                           | Required |
+| --------- | --------------- | ----------- | ------------------------------------- | -------- |
+| `title`   | `string`        | `undefined` | Title of the Header                   |       |
+| `subtitle`   | `string`        | `undefined` | Subtitle of the Header                   |       |
+| `leading` | `React.ReactNode` | `undefined` | leading asset or icon to be placed at the left most side of the ModalHeader |          |
+| `trailing` | `Badge, Link, Text, IconButton`, `undefined` | undefined  |     trailing component to be placed at the right most side of the ModalHeader     |    |
+| `titleSuffix` | `Counter` | `undefined` | A component to be placed adjacent to the title text |          |
+
+### `ModalFooter` API
+> `ModalFooter` API will be similar to [`BottomSheetFooter` API](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/BottomSheet/_decisions/decisions.md#bottomsheetfooter-api).
+
+| Prop       | Type              | Default     | Description                 | Required |
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Contents of the ModalFooter | ✅       |
+
+## Behaviors
+### Modal Stacking
+- We're at consensus that we do need to support modal stacking.
+- Design is not yet aligned on how the modal should look like when stacked and what the motion should be.
+- As an interim solution we can implement a basic stacking behavior with the same exit/entry animations as the current modal and look into enhancing this motion later.
+
+### Modal Sizes & Responsiveness
+- We will offer Modal in 3 sizes - small, medium and large.
+- Each size corresponds to a max-width for the Modal
+- As of today these are the max-widths we're using for each size:
+  - small: `400px`
+  - medium: `760px`
+  - large: `1024px`
+- All the Modals will have a min-width of `320px` regardless of the size.
+- This would mean that the Modal will be responsive and will scale down to `320px` on smaller screens but will not scale up beyond the max-width as per its size mentioned above.
+- POC Link: https://codesandbox.io/s/eloquent-lalande-jvxwno
+
+### Modal Height
+- Each Modal will have a max-height of `80vh` (80% of viewport height) and will be scrollable if the content exceeds this height.
+
+
+### Modal Usage on mWeb & native apps
```

Comment:
all these sections have too many points. Did chatgpt wrote it for you? seems repetitive and have overlaps. can we simplify this and combine the points so we can remove unnecessary points? Like for example

```
- We will not be using the Modal component on mWeb or native apps.
- We will only use the BottomSheet component on mWeb and native apps.
```

could only be one single point
```
- We will not be exposing the Modal component on mWeb or native apps(this is a design guideline) and as a preference BottmSheet shall be use on mobile platforms(mweb and RN)
```